basiclti/basiclti-common/pom.xml

```diff
@@ -33,6 +33,7 @@
         <dependency>
             <groupId>org.sakaiproject.kernel</groupId>
             <artifactId>sakai-kernel-util</artifactId>
+            <!-- Not provided - ResourceLoader must be in the webapp -->
```

Comment:
All this means is that the dependency was declared already and it's not being overridden
```suggestion
```